src/_includes/compass_teaser_for_blog_posts.liquid

```diff
@@ -0,0 +1,8 @@
+<div class="blog-post-compass-teaser">
+  {%- assign anchor_id = site.translations[site.lang].compass-teaser.anchor-id -%}
+  {% include page_location_anchor.html anchor_id=anchor_id %}
```

Comment:
Not sure if we would need to include a page location anchor on blog posts. 

I think it would be unlikely that we would link someone to the compass cta section of a blog post.

Reply:
Ah yeah!